keyboards/xd75/keymaps/bramver/rules.mk

```diff
@@ -0,0 +1,12 @@
+# Build options
+BACKLIGHT_ENABLE = yes		# Switch LEDs
+MOUSEKEY_ENABLE = yes			# Emulates mouse key using keypresses
+RGBLIGHT_ENABLE = yes 		# LED strips
+EXTRAKEY_ENABLE = yes 		# Use system and audio control key codes
+TAP_DANCE_ENABLE = no 		# Use multi-tap features
+AUDIO_ENABLE = no  				# Audio stuff
+UNICODEMAP_ENABLE = yes  	# Emojify me pls
+
+ifndef QUANTUM_DIR
```

Comment:
Could you remove this (ifndef/include/endif) block? 

It's not needed, and is a holdover from the old build system.